test/20.pass/test_make_heap_alloc.py

```diff
@@ -0,0 +1,37 @@
+import freetensor as ft
+
+
+def test_basic():
+    with ft.VarDef([("x", (), "int32", "cache", "cpu/heap"),
+                    ("i", (), "int32", "input", "cpu"),
+                    ("o", (), "int32", "output", "cpu")]) as (x, i, o):
+        with ft.VarDef("t", (), "int32", "cache", "cpu") as t:
+            t[()] = i[()] * 7
+            x[()] = t[()] + 2
+        o[()] = x[()] * 2
+        o[()] *= 2
+    ast = ft.pop_ast(verbose=True)
+    ast = ft.lower(ast, verbose=1)
```

Comment:
You need to construct a more complex example, or disable `prop_one_time_use` pass in `lower`'s `skip_passes` parameter, so the variable you are tested won't be optimized out.